plugins/flytekit-papermill/flytekitplugins/papermill/task.py

```diff
@@ -287,6 +289,8 @@ def execute(self, **kwargs) -> Any:
             else:
                 raise TypeError(f"Expected output {k} of type {type_v} not found in the notebook outputs")
 
+        if len(output_list) == 1:
```

Comment:
what happens if we don't do this?

Reply:
There will be a mismatch between the output type and the downstream task's input type.